ansible_collections/arista/avd/docs/release-notes/4.x.x.md

```diff
@@ -83,15 +83,17 @@ See details in the [Porting guide for AVD 4.x.x](../porting-guides/4.x.x.md#fabr
 ### Change in eos_designs behavior in core_interfaces variables
 
 - `core_interfaces.p2p_links.[].ptp_enable` requires the `ptp.enabled: true` to be set at the fabric level.
```

Comment:
confusing that the previous change now refers to the old key. Let's update that. Maybe also switch the order of these.

Reply:
Copy/paste error. Now fixed